lesson_10/test_check_logs.py

```diff
@@ -0,0 +1,44 @@
+"""Tests for checking logs with login actions and errors."""
+
+import logging
+
+from homework_10 import log_event
+
+LOG_FILE = 'login_system.log'
+logger = logging.getLogger('log_event')
+
+
+def test_log_success_event():
+    """Test for successful login with INFO log."""
+    log_event('user123', 'success')
+    with open(LOG_FILE, 'r') as log_file:
+        log_content = log_file.read()
+    assert ('INFO - Login event - Username: user123, Status: success'
```

Comment:
For these messages you could organize some dict on the top of the module 
Also after assert check better to add some error message in case of failure